packages/babel-helper-module-transforms/src/index.js

```diff
@@ -289,11 +257,18 @@ function buildExportInitializationStatements(
   for (const data of metadata.source.values()) {
     for (const [exportName, importName] of data.reexports) {
       initStatements.push(
-        reexportGetter({
-          EXPORTS: t.identifier(metadata.exportName),
-          EXPORT_NAME: t.stringLiteral(exportName),
-          NAMESPACE: t.identifier(data.name),
-          IMPORT_NAME: t.identifier(importName),
+        template`
+          Object.defineProperty(EXPORTS, "EXPORT_NAME", {
```

Comment:
I could also opt to use `template.ast` on a lot of these and do

```
Object.defineProperty(${metadata.exportName}, "${exportName}", {
```

but I kind of felt like it hurt readability of some of the templates.